docs/connector-development/connector-builder-ui/maintenance.md

```diff
@@ -0,0 +1,175 @@
+# Maintenance through Connector Builder
+
+This document will describe the process through which you can maintain a connector publicly available though our GitHub repository. This process has been split up in five broad steps:
+* Build
+* Merge into code
+* Validation (CATs)
+* Release
+* Operate/Maintain
+
+The visualization of this process is loosely maintained [here](https://whimsical.com/connector-maintenance-06-23-Rh2xAX5Byw641Xpf7P7qNg@2bsEvpTYSt1Hiz1WXyhZwMwZTwhXRsM2Sv8).
+
+## Build
+
+Goal: Modify the behavior of the connector
+
+Expected output: A manifest.yaml file that represents this new behavior
+
+All the steps mentioned below assume that you have the API available to perform requests. 
+
+### Validate Compatibility of Added Features
+Before implementing anything, we need to know if the features you want to add are supported by the Connector Builder. Please refer to the [compatibility guide](./connector-builder-compatibility.md) to confirm if you can build this change within the Connector Builder.
+
+### Config in Connector Builder
+
+Given that the connector already exists and that you want to use the account configured for CATs to iterate during the build phase, you can use [GSM](https://console.cloud.google.com/security/secret-manager?project=dataline-integration-testing) to fetch the existing configuration. Simply write the connector name in the filter bar, click on `SECRET_SOURCE-<name>__CREDS` and under `Actions`, select `View secret value` for the enabled version.
```

Comment:
why not use the `ci_credentials` tool? it seems like annoying and error prone than playing with GSM directly

Reply:
Again, this requires a user to be well versed in Python which is not the target for this document